staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -978,6 +978,22 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 		vmSetName := az.mapLoadBalancerNameToVMSet(lbName, clusterName)
 		// Etag would be changed when updating backend pools, so invalidate lbCache after it.
 		defer az.lbCache.Delete(lbName)
```

Comment:
do we want to invalidate cache? I think we still should since we updated LB before, but i'm not sure why the invalidate happened here instead of L930 once we decide it's a dirtly LB.

Reply:
if it is not a dirty LB, will the etag be changed? @feiskyer ?

Reply:
yep, etag would change when updating backend address pools.

@yangl900 for your question, we have already handled dirty caches inside `az.CreateOrUpdateLB()`